app/helpers/thredded/urls_helper.rb

```diff
@@ -58,5 +58,15 @@ def edit_post_path(post)
         edit_messageboard_topic_post_path(messageboard, post.postable, post)
       end
     end
+
+    # @param post [Post, PrivatePost]
+    # @return [String] path to the DELETE PATCH PUT POST endpoint.
+    def delete_post_path(post)
```

Comment:
I think he tried to do that but ran into some issues with the helper we have as `post_path` now. @TheDudeWithTheThing was that the case?